### PR TITLE
Initial siesta conda-forge

### PR DESCRIPTION
**Pull request by zerothi**
Signed-off-by: Nick Papior <nickpapior@gmail.com>

### CODE REVIEW

File: recipes/siesta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+{% set blas_variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://launchpad.net/{{ name }}/4.0/{{ version }}/+download/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
```
Comment (CJ-Wright) [inline recipes/siesta/meta.yaml]: Would it be possible to sha256?

Comment (zerothi) [reply recipes/siesta/meta.yaml]: The launchpad tar's are intrinsically added a md5, although I could download and get sha256, is it really beneficial?

**Comment by zerothi** [comment]
@CJ-Wright would it be ok to test the 4.0.1 release with md5 sum.

I have just updated the siesta suite to also ship the sha256/sha512 information for new releases.
I.e. all newer releases will use sha256. :)

**Comment by zerothi** [comment]
@CJ-Wright, could you help me with the further merging of this recipe? Many thanks! :)

**Pull request merged**